packages/client/components/PreferredName.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled'
+
+const PreferredName = styled('div')({
```

Comment:
I see we import this component but override most of props. 

For example here: https://github.com/ParabolInc/parabol/pull/7425/files#diff-4f66bc73bb9d80ee94155a5e5adba61a44881f2ba7cea2fb7a9c47bdc8e92570R9-R15

This does not feel generic enough to be a separate component.

Lets better remove this component and specify all props locally.